cognite/client/data_classes/workflows.py

```diff
@@ -232,6 +239,47 @@ def dump(self, camel_case: bool = True) -> dict[str, Any]:
         return output
 
 
+class SimulationTaskParameters(WorkflowTaskParameters):
+    """
+    The simulation parameters are used to specify the simulation routine to be executed.
+    Args:
+        routine_external_id (str): The external ID of the simulation routine to be executed.
+        run_time (int | None): Reference timestamp used for data pre-processing and data sampling.
+        inputs (list[SimulatorRoutineInputConstant | SimulatorRoutineInputTimeseries] | None): List of input overrides
```

Comment:
@lpereiracgn here inputs should be just (`referenceId`, `value` and `unit`) according to the API spec, but I'm seeing that `SimulatorRoutineInputConstant` and `SimulatorRoutineInputTimeseries` are the complete input definitions that we expect when defining a routine.

<img width="647" alt="image" src="https://github.com/user-attachments/assets/ce2b28be-7744-4613-afbd-705f224a5d8f">

Reply:
This input type is the same one as the one required by the simulation run method
<img width="1079" alt="image" src="https://github.com/user-attachments/assets/296ab9ed-53fe-4f0f-b63c-b3095c4111ff">

Reply:
fixed this